.github/workflows/tests.yaml

```diff
@@ -52,6 +53,11 @@ jobs:
       run: |
         git config --global user.email "dvctester@example.com"
         git config --global user.name "DVC Tester"
+    - name: setup gs creds
+      shell: bash
+      run: |
+        mkdir -p scripts/ci
```

Comment:
Using what is already used in the code for now. Don't want to mess with the tests themselves in this PR, just making this run the same way it did on travis.